Papyrus/Source/PapyrusStore.swift

```diff
@@ -55,7 +64,11 @@ public final class PapyrusStore {
         do {
             try self.createDirectoryIfNeeded(at: self.url)
         } catch {
+            #if !os(Linux) && !os(Android) && !os(Windows)
```

Comment:
I wonder whether it is cleaner to bring these checks inside the Logger class. So the Logger class decides whether to print or use os log?

Reply:
If I remember correctly, Linux, Android, and Windows don't have `os.log`, so it can't go directly in the logger. I'm not sure if those OS's have an equivalent logger